app/models/usps_confirmation_code.rb

```diff
@@ -0,0 +1,16 @@
+class UspsConfirmationCode < ApplicationRecord
+  belongs_to :profile
+
+  def self.find_with_otp(otp)
+    find do |usps_confirmation_code|
```

Comment:
I'm super unfamiliar with `find` with a block --- is this a table scan?

Reply:
Oh I see this is used on a relation -- makes wayyyy more sense, but still used the wrong way could be a table scan.

Do you think it would be clearer if we made this a scope? Same result but hopefully it would make it more obvious to users that it's suppsed to be combined with other scopes and *not* just be a finder mehtod like our others (that are used for point queries)

```ruby
scope :first_with_otp, ->(otp) { to_a.find { |usps_confirmation_code| ... } }
```

Reply:
Good idea :+1:

Reply:
Hmm, this is turning out to be problematic since scopes have to return a relation and `find` returns a record 🤔

Reply:
Done ✅